meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -10,8 +10,9 @@ SRC_URI:append:lmp = " \
     file://tmpfiles.conf \
     "
 
-PACKAGECONFIG += "${@bb.utils.filter('MACHINE_FEATURES', 'fiovb', d)} libfyaml"
+PACKAGECONFIG += "${@bb.utils.filter('MACHINE_FEATURES', 'fiovb', d)} ${@bb.utils.filter('MACHINE_FEATURES', 'fioefi', d)} libfyaml"
 PACKAGECONFIG[fiovb] = ",,,optee-fiovb aktualizr-fiovb-env-rollback"
+PACKAGECONFIG[fioefi] = ",,,fioefi aktualizr-fioefi-env-rollback"
```

Comment:
As this new `fioefi` is mutually exclusive with the other `fiovb` we can add a conflict  
```
PACKAGECONFIG[fiovb] = ",,,optee-fiovb aktualizr-fiovb-env-rollback,fioefi"
PACKAGECONFIG[fioefi] = ",,,fioefi aktualizr-fioefi-env-rollback,fiovb"
```
This will protect the user to choose the two

Reply:
Yes, right, but I think it should be:
```
PACKAGECONFIG[fiovb] = ",,,optee-fiovb aktualizr-fiovb-env-rollback,,fioefi"
PACKAGECONFIG[fioefi] = ",,,fioefi aktualizr-fioefi-env-rollback,,fiovb"
```

Reply:
from documentation:
```
PACKAGECONFIG ??= "f1 f2 f3 ..."
     PACKAGECONFIG[f1] = "\
                          --with-f1, \
                          --without-f1, \
                          build-deps-for-f1, \
                          runtime-deps-for-f1, \
                          runtime-recommends-for-f1, \
                          packageconfig-conflicts-for-f1 \
                          "
```

Reply:
right, one more comma is needed as you mention